<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizGroupView</class>
 <widget class="QFrame" name="QuizGroupView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>354</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0" colspan="15">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="14">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>7</width>
          <height>44</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="8">
       <widget class="QToolButton" name="tbtMoveUp">
        <property name="text">
         <string>Выше</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/up.png</normaloff>:/ico/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QToolButton" name="tbtMoveDown">
        <property name="text">
         <string>Ниже</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/down.png</normaloff>:/ico/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="tbtUpLevel">
        <property name="text">
         <string>Наверх</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/view-refresh.png</normaloff>:/ico/view-refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QToolButton" name="tbtProperties">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Свойства</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/edit-find-replace.png</normaloff>:/ico/edit-find-replace.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QToolButton" name="tbtRemove">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/edit-delete.png</normaloff>:/ico/edit-delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="tbtAddRandom">
        <property name="text">
         <string>Выбор</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/bookmark-new.png</normaloff>:/ico/bookmark-new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="tbtAddShuffle">
        <property name="text">
         <string>Перем.</string>
        </property>
        <property name="icon">
         <iconset resource="../../../r.qrc">
          <normaloff>:/ico/contact-new.png</normaloff>:/ico/contact-new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../r.qrc"/>
 </resources>
 <connections/>
</ui>
